manifest.ini

```diff
@@ -3,26 +3,26 @@ server_url = https://updates-autopush.stage.mozaws.net
 push_server = wss://autopush.stage.mozaws.net
 router = stage.autopush.routerv2
 storage = stage.autopush.storage
-bugzilla_api = https://bugzilla-dev.allizom.org
-bugzilla_user = chartjes@mozilla.com
-bugzilla_api_key = oelpF8y4nHYuYf3xOxll4XQXTJtPAkIwrs1BmHvO 
+bugzilla_api =
+bugzilla_user =
+bugzilla_api_key =
 
 
 [production]
 server_url = https://updates.push.services.mozilla.com
```

Comment:
we eventually want to pass all of these URLs dynamically from ops Jenkins to our Jenkins so these will also likely be passed in as ENV vars.  We could always do this later when we wire these tests up w/ CI

Reply:
I don't pass them in dynamically for Kinto -- we pass in the stack we are testing (dist, settings, webextensions) and then target testing environment (stage, production)